packages/office-addin-sso/src/authRoute.ts

```diff
@@ -7,6 +7,7 @@
 import * as express from 'express';
 import * as fetch from 'node-fetch';
 import * as form from 'form-urlencoded';
+import * as usageDataHelper from './usagedata-helper';
```

Comment:
What is usagedata-helper? Is this a dependency in package.json? Why not just `import * as usageData from `office-addin-usage-data`? Also, you don't need `./` here.

Reply:
usagedataHelper is intended as a wrapper for office-addin-usage data methods. I implemented it because I felt it de-cluttered the code a bit and made things neater. Otherwise I have to fashion an object each time I was to send a new success event

Reply:
I think it points out that the underlying methods exposed weren't designed to be used as easily as they could. But even with the helper, I don't think you should be importing the same file across the other packages. It breaks the modular design. It would be better to just make a local helper method for now, but we should really discuss a better pattern for this in the future. Perhaps we can chat over Teams as this may not fully capture my point.